src/simpleModels/ServiceAddress.php

```diff
@@ -22,7 +22,7 @@ class ServiceAddress {
         "AddressLine2" => array("type" => "string"),
         "PlusFour" => array("type" => "string"),
         "AddressType" => array("type" => "string"),
-        "UnparsedAddress" => array("type" => "string")
+        "UnparsedAddress" => array("type" => "string") // Note that UnparsedAddress is Read-Only!
```

Comment:
Good coding practice is to _not comment your code_ but if you do put it _before the line of code_ under context